src/azure-cli/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -88,13 +89,15 @@ def load_command_table(self, _):
                          validator=validate_private_endpoint_connection_id)
         g.command('delete', 'delete', validator=validate_private_endpoint_connection_id)
         g.show_command('show', 'get', validator=validate_private_endpoint_connection_id)
+        g.wait_command('wait', validator=validate_private_endpoint_connection_id)
```

Comment:
You have an additional wait command here, but I don't find you have `support_no_wait` for approve/reject/delete. Is there anything I missed?

Reply:
> You have an additional wait command here, but I don't find you have `support_no_wait` for approve/reject/delete. Is there anything I missed?

Nice catch, I noticed this just after I submitted the PR... I will add `support_no_wait` for `approve` and `reject`.